monitoring/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM node:14-alpine as builder
+
+# Add packages necessary to perform the build process.
+RUN apk add --update --no-cache \
+    git \
+    bash \
+    build-base \
+    python3 \
+    && \
+    rm -rf /var/cache/apk/ && mkdir /var/cache/apk/ && \
+    rm -rf /usr/share/man
+
+# Install node-prune (https://github.com/tj/node-prune)
+RUN wget https://gobinaries.com/tj/node-prune
+RUN sh node-prune
+
+# Use `https://` instead of unauthenticated `git://` protocol.
+RUN git config --global url."https://".insteadOf git://
+
+WORKDIR /monitoring-build
+
+# Copy only essential files.
+COPY package.json yarn.lock ./
+COPY tsconfig.json ./
+COPY src ./src
+
+RUN yarn install --frozen-lockfile --ignore-scripts
+RUN yarn run postinstall
+
+RUN yarn build
+
+# Prune development dependencies.
+RUN rm -rf ./node_modules
+RUN rm -rf ./external
+RUN yarn install --production --frozen-lockfile --ignore-scripts
+RUN yarn run postinstall
+
+# Prune other unnecessary files from dependencies using node-prune.
+RUN node-prune
```

Comment:
Out of curiosity what's the image size reduction we achieved thanks to the `node-prune`?

Reply:
IIRC smth around 300MB